documentation/kyverno-cli.md

```diff
@@ -40,39 +39,58 @@ yay -S kyverno-git
 
 Prints the version of kyverno used by the CLI.
 
-Example: 
+Example:
+
 ```
 kyverno version
 ```
 
 #### Validate
-Validates a policy, can validate multiple policy resource description files or even an entire folder containing policy resource description 
+
+Validates a policy, can validate multiple policy resource description files or even an entire folder containing policy resource description
 files. Currently supports files with resource description in yaml.
 
 Example:
+
 ```
 kyverno validate /path/to/policy1.yaml /path/to/policy2.yaml /path/to/folderFullOfPolicies
 ```
 
 #### Apply
+
 Applies policies on resources, and supports applying multiple policies on multiple resources in a single command.
 Also supports applying the given policies to an entire cluster. The current kubectl context will be used to access the cluster.
- Will return results to stdout.
+Will return results to stdout.
 
 Apply to a resource:
-```
+
+```bash
 kyverno apply /path/to/policy.yaml --resource /path/to/resource.yaml
 ```
 
 Apply to all matching resources in a cluster:
-```
+
+```bash
 kyverno apply /path/to/policy.yaml --cluster > policy-results.txt
 ```
 
 Apply multiple policies to multiple resources:
-```
+
+```bash
 kyverno apply /path/to/policy1.yaml /path/to/folderFullOfPolicies --resource /path/to/resource1.yaml --resource /path/to/resource2.yaml --cluster
 ```
 
+##### Exit Codes
+
+When CLI executes successfully, it returns exit code 0.
+
+The CLI exits with diffenent exit codes depending on the type of error:
+
+| Message                                        | Exit Code |
+| ---------------------------------------------- | --------- |
+| Failed to validate atleast one of the Policies | 3         |
+| Failed to apply Mutation                       | 4         |
```

Comment:
How will this work if a policy has a combination of validate and mutate rules? Also, how will be report if a policy is invalid? My suggestion is to simplify to:
0 - success (policies are valid and were applied)
1 - one or more policy rules are violated
2 - invalid policy syntax

Reply:
I have updated the code accordingly.